grpc/proto/search_get_v1.proto

```diff
@@ -0,0 +1,267 @@
+  // extendable in the future
+  // protolint:disable:next REPEATED_FIELD_NAMES_PLURALIZED
+  repeated string path = 2;
+}
+
+message GenerativeSearch {
+  string single_response_prompt = 1;
+  string grouped_response_task = 2;
+  repeated string grouped_properties = 3;
+}
+
+message TextArray {
+  repeated string values = 1;
+}
+message IntArray {
+  repeated int64 values = 1;
+}
+message NumberArray {
+  repeated double values = 1;
+}
+message BooleanArray {
+  repeated bool values = 1;
+}
+
+message Filters {
+  enum Operator {
+    OPERATOR_UNSPECIFIED = 0;
+    OPERATOR_EQUAL = 1;
+    OPERATOR_NOT_EQUAL = 2;
+    OPERATOR_GREATER_THAN = 3;
+    OPERATOR_GREATER_THAN_EQUAL = 4;
+    OPERATOR_LESS_THAN = 5;
+    OPERATOR_LESS_THAN_EQUAL = 6;
+    OPERATOR_AND = 7;
+    OPERATOR_OR = 8;
+    OPERATOR_WITHIN_GEO_RANGE = 9;
+    OPERATOR_LIKE = 10;
+    OPERATOR_IS_NULL = 11;
+    OPERATOR_CONTAINS_ANY = 12;
+    OPERATOR_CONTAINS_ALL = 13;
+  }
+
+
+  Operator operator = 1;
+  // protolint:disable:next REPEATED_FIELD_NAMES_PLURALIZED
+  repeated string on = 2;
+  repeated Filters filters = 3;
+  oneof test_value{
+    string value_text = 4;
+    int64 value_int = 5;
+    bool value_boolean = 6;
+    float value_number = 7;
+    TextArray value_text_array = 9;
+    IntArray value_int_array = 10;
+    BooleanArray value_boolean_array = 11;
+    NumberArray value_number_array = 12;
+  };
+}
+
+
+message MetadataRequest {
+  bool uuid = 1;
+  bool vector = 2;
+  bool creation_time_unix = 3;
+  bool last_update_time_unix = 4;
+  bool distance = 5;
+  bool certainty = 6;
+  bool score = 7;
+  bool explain_score = 8;
+  bool is_consistent = 9;
+}
+
+
+message PropertiesRequest {
+  repeated string non_ref_properties = 1;
+  repeated RefPropertiesRequest ref_properties = 2;
+}
+
+message Hybrid {
+  string query = 1;
+  repeated string properties = 2;
+  // protolint:disable:next REPEATED_FIELD_NAMES_PLURALIZED
+  repeated float vector = 3;
+  float alpha = 4;
+  enum FusionType {
+    FUSION_TYPE_UNSPECIFIED = 0;
+    FUSION_TYPE_RANKED = 1;
+    FUSION_TYPE_RELATIVE_SCORE = 2;
+  }
+  FusionType fusion_type = 5;
+}
+
+message NearTextSearch {
+  message Move{
+    float force = 1;
+    repeated string concepts = 2;
+    repeated string uuids = 3;
+  }
+
+  // protolint:disable:next REPEATED_FIELD_NAMES_PLURALIZED
+  repeated string query = 1;
+  optional double certainty = 2;
+  optional double distance = 3;
+  optional Move move_to = 4;
+  optional Move move_away = 5;
+};
+
+message NearImageSearch {
+  string image = 1;
+  optional double certainty = 2;
+  optional double distance = 3;
+};
+
+message NearAudioSearch {
+  string audio = 1;
+  optional double certainty = 2;
+  optional double distance = 3;
+};
+
+message NearVideoSearch {
+  string video = 1;
+  optional double certainty = 2;
+  optional double distance = 3;
+};
+
+message BM25 {
+  string query = 1;
+  repeated string properties = 2;
+}
+
+
```

Comment:
sometimes there are 2 empty lines, sometimes 1 empty line, can we refactor this also to be always 1 empty line?